src/components/questions/modals/AnswerModal.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Modal from '@material-ui/core/Modal';
+import Backdrop from '@material-ui/core/Backdrop';
+import Fade from '@material-ui/core/Fade';
+
+
+import AnswerForm from './AnswerForm.js';
+
+const useStyles = makeStyles(theme => ({
+    modal: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+
+    },
+    paper: {
+        backgroundColor: theme.palette.background.paper,
+        border: '2px solid #000',
+        boxShadow: theme.shadows[5],
+        padding: theme.spacing(4, 10, 6),
```

Comment:
This use of theme isn't something I came across when looking at `makeStyles` docs, I'm going to have to look into it and incorporate it into my styles.

Reply:
Up to you! I can delete it if we don't want to use it

Reply:
No, it's completely fine! I just didn't even realize the functionality existed!